Lesson5/src/app/Layouts/Layout.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Menu from '../components/Menu';
+import MenuItem from '../components/MenuItem';
+
+export default class Layout extends React.Component{
+	constructor(props){
+		super();
+		this.brand = "React blog!";
+	}
+	isActive(href){
+		return window.location.pathname === href;
+	}
+	
+	render(){
+		return(
+			<>
+			<Menu brand={this.brand}>
+				<MenuItem href="/" active = {this.isActive('/')}> Главная  </MenuItem>
+				<MenuItem href="users" active = {this.isActive('/users')}>Пользователи</MenuItem>
```

Comment:
```suggestion
				<MenuItem href="/users" active = {this.isActive('/users')}>Пользователи</MenuItem>
```
и везде лучше сделать так, чтобы ссылки работали правильно